serverless.yml

```diff
@@ -90,23 +90,35 @@ layers:
 
 functions:
   onConnect:
-    handler: src/handler.onConnect
+    handler: src/functions/onConnect.handler
```

Comment:
Not sure if lambda function will contain all the other files (ondisconnect, onconnect) in /functions if done this way. You can deploy and see if this is the case.

Reply:
It will package all the files in src/ dir, see `package` block in serverless.yml. If you can package individual lambda function or the functions/ dir, then you need to add everything else (e.g. db.ts, types.ts etc.) to a lambda layer which is kinda painful to do, so I am going to keep this structure for now. The file size seems to be ok (24.1kB).